src/engine/renderer/gl_shader.cpp

```diff
@@ -1453,8 +1459,11 @@ GLuint GLShaderManager::CompileShader( Str::StringRef programName,
 
 	if ( !compiled )
 	{
-		PrintShaderSource( programName, shader );
-		PrintInfoLog( shader );
+		std::string log = GetInfoLog( shader );
+		std::vector<InfoLogPair> infoLog = ParseInfoLog( log );
+		PrintShaderSource( programName, shader, infoLog );
+		Log::Warn( "Compile log:" );
```

Comment:
`"Compile log:\n%s"` would be better, so that the first line of the compile log doesn't have `Warn:` and yellow text while the other ones don't.

Reply:
Good point, fixed.